src/components/UniversalNavbar/UniversalNavbar.js

```diff
@@ -108,16 +117,16 @@ class UniversalNavbar extends React.Component {
                     showMobileMenu ? 'shownMobileMenu' : 'hiddenMobileMenu'
                   }
                 >
-                  <a href="/">
+                  <Link href="/">
                     {LogoWhite({ className: 'universal-navbar-logo' })}
-                  </a>
+                  </Link>
                   <Spacer.H56 />
                   {navbarLinks.map((l) => (
                     <div key={l.title + 'mobile'} style={{ marginBottom: 24 }}>
                       <TitleXLarge.Sans.Regular400>
-                        <a key={l.title + 'mobile'} href={l.href}>
+                        <Link key={l.title + 'mobile'} href={l.href}>
```

Comment:
What if two links have same title? I think this would break. See `src/components/Inputs/ButtonSelectGroup/OptionButton.js` which I believe is using uuid for aria labels, but, since we already have that dep you can use for keys too. I did this somewhere in CMS recently if you're looking for an example